worker/worker.ts

```diff
@@ -0,0 +1,53 @@
+import { Worker, Job } from 'bullmq';
+import { redisConnection } from '../shared/config';
+
+export interface JobWorker {
```

Comment:
this interface doesn't appear to ever be used.  do we need an abstract class and an interface?